urgent-care-tests/pom.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
-    <artifactId>health-apis-parent</artifactId>
     <groupId>gov.va.api.health</groupId>
+    <artifactId>test-starter</artifactId>
```

Comment:
Why are you pulling in test-starter exactly? I haven't used it before so I don't know what it's offering. I see that it redefines some docker packaging but I don't know what the rest of it is doing for us.